pkgs/applications/misc/calibre/default.nix

```diff
@@ -92,6 +92,10 @@ stdenv.mkDerivation rec {
     for entry in $out/share/applications/*.desktop; do
       substituteAllInPlace $entry
     done
+
+    for entry in $out/lib/calibre/calibre/plugins/*.so; do
```

Comment:
can you add a comment explaining why you're doing this and refer from there to this issue